attributes/default.rb

```diff
@@ -137,6 +137,7 @@
   # as 'interval' if you want chef-client to be run
   # periodically by launchd
   default['chef_client']['launchd_mode'] = 'interval'
+  default['chef_client']['launchd_user'] = 'root'
```

Comment:
I think we can just hardcode `'root'` in the recipe, or use the `dir_owner` and `dir_group` helper methods in the cookbook.